### PR TITLE
Automatically deploy api branch to alpha

### PR DESCRIPTION
**Pull request by andyhd**
## What

Add Circle CI deploy step to deploy to alpha cluster if branch name is `api`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -62,10 +62,21 @@ jobs:
...
             curl ${JENKINS_URL}/job/control-panel/job/api/buildWithParameters \
               --data token=${JENKINS_JOB_TRIGGER_TOKEN} \
               --data COMMIT_HASH=${CIRCLE_SHA1} \
               --data BRANCH_NAME=${CIRCLE_BRANCH} \
               --user ${JENKINS_USER}:${JENKINS_TOKEN}
+
+      - deploy:
+          name: Deploy to alpha
+          command: |
+            if [ "$CIRCLE_BRANCH}" == "api" ]; then
```
Comment (jmoz) [inline .circleci/config.yml]: Missing open bracket?  `$CIRCLE_BRANCH}`

Comment (andyhd) [reply .circleci/config.yml]: Good spot, thanks

File: .circleci/config.yml [hunk 2]
```diff
@@ -62,10 +62,21 @@ jobs:
...
             curl ${JENKINS_URL}/job/control-panel/job/api/buildWithParameters \
               --data token=${JENKINS_JOB_TRIGGER_TOKEN} \
               --data COMMIT_HASH=${CIRCLE_SHA1} \
               --data BRANCH_NAME=${CIRCLE_BRANCH} \
               --user ${JENKINS_USER}:${JENKINS_TOKEN}
+
+      - deploy:
+          name: Deploy to alpha
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "api" ]; then
```
Comment (xoen) [inline .circleci/config.yml]: I was about to ask if there was a `branch:` field to use instead...

But looking at CircleCI documentation looks like that's how they're doing it there.

**Comment by xoen** [comment]
Looks good 👍

**Pull request merged**